src/components/dialogs/ConfirmDissolution.vue

```diff
@@ -56,28 +57,64 @@ export default class ConfirmDissolution extends Vue {
 
   // Global getters
   @Getter getEntityName!: string
-  @Getter isCoop!: boolean
+  @Getter getEntityType!: string
 
   /** The entity title to display. */
   private get entityTitle (): string {
-    return this.isCoop ? 'Cooperative Association' : 'Company'
+    return this.getEntityDetails().entityTitle
   }
 
   /** The entity title to display. */
   private get subEntityTitle (): string {
-    return this.isCoop ? 'cooperative' : 'company'
+    return this.getEntityDetails().subTitle
   }
 
   /** The entity title to display. */
   private get entityAct (): string {
-    return this.isCoop ? 'Cooperative Association' : 'Business Corporations'
+    return this.getEntityDetails().act
+  }
+
+  /** The entity title to display. */
+  private get getModalTitle (): string {
+    return this.getEntityDetails().title
   }
 
   // Pass click event to parent.
   @Emit() private close () { }
 
   // Pass click event to parent.
   @Emit() private proceed () { }
+
+  // returning  title subtitle, and act by entityType
+  private getEntityDetails (): any {
```

Comment:
Did you try this as a getter (since there are no formal parameters)? It would probably have been reactive to the entity type. It might have simplified some syntax, and as a plug, getters are cached so often the code is faster :)